src/client/event_handler.rs

```diff
@@ -393,7 +392,7 @@ pub trait EventHandler: Send + Sync {
     /// Dispatched when an unknown event was sent from discord.
     ///
     /// Provides the event's name and its unparsed data.
-    async fn unknown(&self, _ctx: Context, _name: String, _raw: Value) {}
+    async fn unknown(&self, _ctx: Context) {}
```

Comment:
I don't suppose we can ask serde, at least, to supply us the unknown variant's name? Without any data supplied to this event, it's just unnecessary. What useful behaviour can you put here, even? Just a log saying, "hey! I just encountered an unknown event. not sure what it is, though"?

If not, maybe it would be good to remove this handler, since without either the name or the unparsed data, the documentation comment is outdated.